app/src/main/java/com/infomaniak/swisstransfer/ui/screen/newtransfer/importfiles/ImportFilesScreen.kt

```diff
@@ -117,8 +128,21 @@ private fun ImportFilesScreen(
                     showUploadSourceChoiceBottomSheet = { showUploadSourceChoiceBottomSheet = true },
                     removeFileByUid = removeFileByUid,
                 )
-                ImportFilesTitle(titleRes = R.string.transferTypeTitle)
-                ImportFilesTitle(titleRes = R.string.advancedSettingsTitle)
+                ImportTextFields(initialShouldShowEmailAddressesFields)
+                ImportFilesTitle(Modifier.padding(vertical = Margin.Medium), titleRes = R.string.transferTypeTitle)
+                TransferTypeButtons(initialSelectedTransferType = TransferType.LINK, onClick = {})
```

Comment:
Your buttons have some kind of padding that cuts them before reaching the edge of the screen. The component should go up to the edge of the screen and take in some form of contentPadding to solve this issue the same way Material components do

Reply:
This should also be in a PR truly dedicated to the ImportScreen

Reply:
I didn't understand that this PR doesn't include the whole screen, then I understand